app/domain/importers/content_details.rb

```diff
@@ -13,8 +13,29 @@ def initialize(content_id, base_path)
 
   def run
     response = items_service.fetch_raw_json(base_path)
-    number_of_pdfs = Performance::Metrics::NumberOfPdfs.parse(response.to_h['details'])
+    metadata = format_metadata(response.to_h)
+    attributes =  metadata.merge(raw_json: response)
     item = Dimensions::Item.find_by(content_id: content_id, latest: true)
-    item.update_attributes(raw_json: response, number_of_pdfs: number_of_pdfs)
+    item.update_attributes(attributes)
+  end
+
+private
+
+  def format_metadata(formatted_response)
+    formatted_response.slice(
+      'content_id',
+      'title',
+      'document_type',
+      'content_purpose_supertype',
+      'first_published_at',
+      'public_updated_at',
+    ).merge(
+      'last_updated_time' => formatted_response['updated_at'],
```

Comment:
Could you please check that `last_update_time` correspond with `updated_at`. This does not feel right to me. They are different dates.